utils/https.h

```diff
@@ -9,6 +9,7 @@
 #ifndef UTILS_HTTPS_H
 #define UTILS_HTTPS_H
 
+#include <stdio.h>
```

Comment:
Why did you include `<stdio.h>`?

Reply:
size_t

Reply:
No, you should include `<stddef.h>`. Always avoid unnecessary header inclusion.